Tools/scripts/verify-ensurepip-wheels.py

```diff
@@ -0,0 +1,78 @@
+"""Compares checksums for wheels in :mod:`ensurepip` against the Cheeseshop."""
+
+import hashlib
+import json
+from pathlib import Path
+import re
+from urllib.request import urlopen
+
+PACKAGE_NAMES = ("pip", "setuptools")
+ROOT = Path(__file__).parent.parent.parent / "Lib/ensurepip"
+WHEEL_DIR = ROOT / "_bundled"
+ENSURE_PIP_INIT_PY_TEXT = (ROOT / "__init__.py").read_text(encoding="utf-8")
+
+# Contains names of successfully verified packages
+verified_packages = {*()}
+
+
+def error(file, message):
+    print(f"::error file={file}::{message}\n")
+
+
+for package_name in PACKAGE_NAMES:
+    # Find the package on disk
+    package_path = next(WHEEL_DIR.glob(f"{package_name}*.whl"), None)
+    if package_path is None:
+        continue
+
+    print(f"Verifying checksum for {package_path}.")
+
+    # Find the version of the package used by ensurepip
+    package_version_match = re.search(
+        f'_{package_name.upper()}_VERSION = "([^"]+)',
+        ENSURE_PIP_INIT_PY_TEXT
+    )
+    if package_version_match is None:
+        error(package_path, f"No {package_name} version found in Lib/ensurepip/__init__.py.")
+        continue
+    package_version = package_version_match[1]
+
+    # Get the SHA 256 digest from the Cheeseshop
+    try:
+        raw_text = urlopen(f"https://pypi.org/pypi/{package_name}/json").read()
+    except (OSError, ValueError) as error:
+        error(package_path, f"Could not fetch JSON metadata for {package_name}.")
+        continue
+
+    expected_digest = ""
+    release_files = json.loads(raw_text)["releases"][package_version]
+    for release_info in release_files:
+        if package_path.name != release_info["filename"]:
+            continue
```

Comment:
```suggestion
            error(package_path, "Wheel name is modified.")
            continue
```

Reply:
This continues the inner for, if an sdist is returned before the wheel then this would trigger.

A